specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/autoDismissAlertsRulesStatistics.json

```diff
@@ -0,0 +1,122 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "Security Center",
+      "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+      "version": "2019-01-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+      "https"
+    ],
+    "consumes": [
+      "application/json"
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "security": [
+      {
+        "azure_auth": [
+          "user_impersonation"
+        ]
+      }
+    ],
+    "securityDefinitions": {
+      "azure_auth": {
+        "type": "oauth2",
+        "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+        "flow": "implicit",
+        "description": "Azure Active Directory OAuth2 Flow",
+        "scopes": {
+          "user_impersonation": "impersonate your user account"
+        }
+      }
+    },
+    "paths": {
+      "/subscriptions/{subscriptionId}/providers/Microsoft.Security/autoDismissAlertsRulesStatistics": {
+        "post": {
+          "x-ms-examples": {
+            "Receive rule statistics": {
+              "$ref": "./examples/AutoDismissAlertsRulesStatistics/PostAutoDismissAlertsRulesStatistics_example.json"
+            }
+          },
+          "tags": [
+            "AutoDismissAlertsRulesStatistics"
+          ],
+          "description": "Receive rule statistics",
```

Comment:
How about something with more details like:
"This API test the given auto-dismiss rule against alert in the given subscription and reruns relevant statistics." also consider taking this with our technical writer